src/ObsidianData.php

```diff
@@ -28,7 +34,7 @@ public function getBlock() : Opaque
 
     public function getPosition() : Position
     {
-        return $this->block->getPosition();
+        return $this->block; // Now you can access $block

```

Comment:
i think it's better just change it like this, maybe the creator forgot to change the variable.
```suggestion
        return $this->obsidian->getPosition();
```

Reply:
maybe, But He can decide however, I tested my version and it worked perfectly fine, No issues, or crashes.